src/main/kotlin/lotto/domain/Lotto.kt

```diff
@@ -1,13 +1,15 @@
-package lotto
+package lotto.domain
 
 class Lotto(val numbers: Set<LottoNumber>) {
```

Comment:
```suggestion
class Lotto(private val numbers: Set<LottoNumber>) {
```
캡슐화를 해볼수 있을까요? https://github.com/next-step/kotlin-lotto/pull/324#discussion_r884151043 와 같이 생각해 봅시다.

> Lotto가 일급컬렉션으로 구현된걸까요 ?

`contains` 라는 함수를 만드신것 처럼 `numbers` 를 공개하지 말고 필요한 맴버함수들을 만든다면 더 완벽해 집니다.
`Set<LottoNumber>` 는 단순히 자료구조이지만 `Lotto` 는 도메인에 의미있는 객체에요. `Set<LottoNumber>` 는 자료구조에 필요한 메소드들만 제공하지만 일급 컬렉션은 도메인 내에 의미있는 메소드들을 제공할 수 있어요.